operations/jasper.go

```diff
@@ -26,24 +34,112 @@ func Jasper() cli.Command {
 		Subcommands: []cli.Command{
 			jasperGRPC(),
 			jasperREST(),
+			jasperCombined(),
 		},
 	}
 }
 
+func jasperCombined() cli.Command {
+	return cli.Command{
+		Name:  "service",
+		Usage: "starts a combined multiprotocol jasper service",
+		Flags: []cli.Flag{
+			cli.IntFlag{
+				Name:   "rpcPort",
+				EnvVar: envVarJasperGRPCPort,
+				Value:  defaultJasperGRPCPort,
+			},
+			cli.StringFlag{
+				Name:   "rpcHost",
+				EnvVar: envVarJasperGRPCHost,
+				Value:  defaultLocalHostName,
+			},
+			cli.IntFlag{
+				Name:   "restPort",
+				EnvVar: envVarJasperGRPCPort,
+				Value:  defaultJasperRESTPort,
+			},
+			cli.StringFlag{
+				Name:   "restHost",
+				EnvVar: envVarJasperRESTHost,
+				Value:  defaultLocalHostName,
+			},
+		},
+		Action: func(c *cli.Context) error {
+			restHost := c.String("restHost")
+			restPort := c.Int("restPort")
+			rpcAddr := fmt.Sprintf("%s:%d", c.String("rpcHost"), c.Int("rpcPort"))
+
+			ctx, cancel := context.WithCancel(context.Background())
+			defer cancel()
+
+			mngr := jasper.NewLocalManagerBlockingProcesses()
+
+			// assemble the rest service
```

Comment:
The "assemble" section seems identical to the existing code in jasperGRPC() and jasperREST(). Is it worth splitting them out into their functions?